src/stdio_mgr/stdio_mgr.py

```diff
@@ -34,14 +34,19 @@
 
 
 class _PersistedBytesIO(BytesIO):
-    """Class to persist the stream after close.
+    """Class to persist the value after close.
 
-    A copy of the bytes value is available at :attr:`_closed_buf` after
+    A copy of the bytes value is given to a callback prior to
     the :meth:`~BytesIO.close`.
     """
 
+    def __init__(self, closure_callback):
```

Comment:
My first reflex was that this should be an optional argument. But, then I realized that it's the whole point of the superclass, and so definitely should be required.

Is too bad the positional-only syntax is so far away from being universal -- would be a great place to use it.  Oh well. Come 2024 or whatever, can change it. :-P

Reply:
This private class will be rewritten many times before 2024.